src/Storage/TSStorageManager.m

```diff
@@ -456,16 +448,15 @@ - (void)deleteDatabaseFile {
     }
 }
 
-- (void)wipeSignalStorage {
+- (void)resetSignalStorage {
     self.database = nil;
+    _dbConnection = nil;
```

Comment:
I prefer property getters wherever possible (even internally). My rationale is that some assumptions become easier when only getters, setters, dealloc, and init touch it.

I could be persuaded to *always* use ivars internally, but I don't think we should mix and match arbitrarily.